### DISCUSSION TITLE
Guidance on Applying for the Student Developer Pack as a PhD Student Using a Career Certificate

### DISCUSSION THREAD

**Discussion by christian-giannetti** [community/community#145334] [GitHub Education]
### Hi everyone,

I am seeking assistance with my application as the process has been somewhat confusing for me. I am currently pursuing a PhD at an Italian university, which means I am affiliated with the institution. However, I have encountered a challenge: the university does not issue student ID cards to PhD students, as these are only provided to undergraduate and master’s students.

For these reasons, I am submitting a career certificate issued by the university, which provides comprehensive evidence of my enrollment and affiliation. It confirms my status as a PhD student for the academic year 2024/2025, includes my personal details (such as student ID and citizenship), specifies the program’s duration, the date of enrollment, and the date the document was issued (today). Additionally, it bears the university’s official logo and is signed by the PhD office.

I suspect that your systems may be misclassifying my document based on certain keywords, interpreting it as a certificate of eligibility for a scholarship, when in reality, it clearly states that I am affiliated with and work for the university.

Despite this, my application has been flagged, and I believe this misclassification may be the cause. I would greatly appreciate your help in clarifying this situation and ensuring that the document is reviewed appropriately. Could you confirm whether this document meets your requirements, or let me know if there are additional steps I need to take to complete my application?

Thank you in advance for your support.

Best regards,
Christian

**Comment by christian-giannetti** [comment]
Up

**Comment by Renoviejo** [comment]
I am having the same problem, did you manage to solve it?
